cookiecutter/vue_template.yaml

```diff
@@ -0,0 +1,3 @@
+default_context:
+  client_app: "Vue3"
+  use_graphql: "n"
```

Comment:
cookiecutter templates. Used for automation here, but you can also use them for local testing so you can skip the questionnaire.